bulbs/special_coverage/templates/special_coverage/bulbs_sc_landing.html

```diff
@@ -0,0 +1,66 @@
+{% load betty humanize ads static %}
+
+<div class="sc-hero">
+  <figure class="hero-img">
+    {% cropped content.thumbnail %}
+  </figure>
+  <div class="sc-label">Special Coverage</div>
+  <div class="billboard">
+    <h1>{{ special_coverage.name }}</h1>
+    {% if special_coverage.description %}
+      <p>{{ special_coverage.description }}</p>
+    {% endif %}
+    <div class="sponsor">
+      <div class="presented-by">
+        <span>Presented by</span>
+        {% if special_coverage.campaign.sponsor_logo %}
```

Comment:
We should coordinate the work on this with everything @kand, @benghaziboy and co have been doing all week as the `campaign` reference directly on special coverage is going away.